routers/web/web.go

```diff
@@ -277,9 +277,15 @@ func RegisterRoutes(m *web.Route) {
 			m.Get("/{provider}", auth.SignInOAuth)
 			m.Get("/{provider}/callback", auth.SignInOAuthCallback)
 		})
-		m.Get("/link_account", auth.LinkAccount)
-		m.Post("/link_account_signin", bindIgnErr(forms.SignInForm{}), auth.LinkAccountPostSignIn)
-		m.Post("/link_account_signup", bindIgnErr(forms.RegisterForm{}), auth.LinkAccountPostRegister)
+		m.Group("/link_account", func() {
+			m.Get("", auth.LinkAccount)
+		}, openIDSignInEnabled)
```

Comment:
`openIDSignInEnabled` can be added to the `Get` and `Post` arguments, no need to go all fancy with `Group` here.

Reply:
Just tested that
```
m.Get("/", Home, openIDSignInEnabled)
```
does not forbid access to / when openIDSignInEnabled is disabled but

```
m.Group("/", func() {
		m.Get("", Home)
	}, openIDSignInEnabled)
```
does. Seems that simply adding `openIDSignInEnabled` to `Get` arguments does not work.

Reply:
```
m.Get("/", openIDSignInEnabled, Home)
```
works ok - will fix it.

Reply:
Done.